content/blog/2019/08/2019-08-17-folder-auth-plugin.adoc

```diff
@@ -0,0 +1,121 @@
+---
+layout: post
+title: "Introducing new Folder Authorization Plugin"
+tags:
+- plugins
+- security
+- performance
+- gsoc
+- gsoc2019
+author: abhyudayasharma
+---
+
+During my link:/projects/gsoc/2019/role-strategy-performance[Google Summer of Code Project],
+I have created the brand new plugin:folder-auth[Folder Auth Plugin] for easily
+managing permissions to projects organized in folders from the plugin:cloudbees-folder[Folders plugin].
+This new plugin is designed for fast permission checks with easy-to-manage roles.
+The 1.0 version of the plugin has just been released and can be downloaded
+from your Jenkins' Update center.
+
+image::/images/post-images/gsoc-folder-auth-plugin/folder-auth.png[Screenshot of the Folder Auth Plugin]
+
+This plugin was inspired by the plugin:role-strategy[Role Strategy Plugin]
+and brings about performance improvements and makes managing roles much easier.
+The plugin was developed to overcome performance limitations of the Role Strategy
+plugin on a large number of roles. At the same time, the plugin addresses one
+of the most popular ways of organizing projects in Jenkins, through folders.
+The plugin also has a new UI with more improvements to come in the future.
+
+The plugin supports three types of roles which are applicable at different places
+in Jenkins.
+
+* Global Roles: applicable everywhere in Jenkins
+* Agent Roles: restrict permissions for multiple agents connected to your instance
+* Folder Roles: applicable to multiple jobs organized inside folders
+
+This plugin, unlike the Role Strategy plugin, does not use regular expressions
```

Comment:
What about moving it to the top of the next section? It would look better IMHO

```
This plugin is designed to outperform Role Strategy Plugin in permission
checks. 
Unlike the Role Strategy plugin, it does not use regular expressions ....

The improvements were measured using the
link:blog/2019/06/21/performance-testing-jenkins/[micro-benchmark framework] ...